### DISCUSSION TITLE
v5?

### DISCUSSION THREAD

**Discussion by dougmoscrop** [aarondcohen/id128#21] [Ideas]
Is there any interest in having a v5 implementation? I can't promise I would do it, but just curious

**Comment by aarondcohen** [comment]
i've considered it.
pros:
- completeness (though v2 seems fully deprecated)
- providing a uniform interface
cons:
- the browser requires a custom sha1 implementation (integrating a properly licensed implementation is a consideration)
- the official uuid v5 library is pretty fast (not a reason)
- more versions mean a heavier library

perhaps a server-only implementation is worth considering.  I'm happy for collaboration.
